web/src/components/Jwt.res

```diff
@@ -0,0 +1,37 @@
+// Copyright (C) 2021 Monocle authors
```

Comment:
nit: perhaps we should stop putting all the code in the components directory. How about we move this and the Store to a `web/src/lib` directory?